composer.json

```diff
@@ -18,10 +18,10 @@
         "doctrine/persistence": "^1.1",
         "gisostallenberg/response-content-negotiation-bundle": "^0.5.1",
         "haydenpierce/class-finder": "^0.4.0",
-        "hwi/oauth-bundle": "^0.6.3",
```

Comment:
Shouldn't this be "^0.6.3 || ^1.0". As it will lead to less conflicts in combination with other packages or implementations, this also counts for the other upgrades, composer will choose the best available options/combination

Reply:
this also counts for the other upgrades ... 

"php-http/guzzle6-adapter": "^1.0 || ^2.0",

And this should stay the same:
"php-http/httplug-bundle": "^1.16",
composer will choose the best available options/combination